include/retdec/pelib/ImageLoader.h

```diff
@@ -202,6 +202,11 @@ class ImageLoader
 		return (sectionIndex < sections.size()) ? &sections[sectionIndex] : nullptr;
```

Comment:
_(A comment not related to the attached line.)_

The following test is failing for this PR, both on Linux as well as on Windows ([Linux build](https://retdec-tc.avast.com/viewLog.html?buildId=5607&buildTypeId=Retdec_LinuxBuild&tab=buildLog&_focus=12149), [Windows build](https://retdec-tc.avast.com/viewLog.html?buildId=5611&buildTypeId=Retdec_WinBuild&tab=buildLog&_focus=24461)):
```
tools.fileinfo.features.max-memory.TestAnalysisWhenSufficient (E556E9A67BBED5B6C32BE434A14884[..])  [ FAIL ]  (0.14s)
F
======================================================================
FAIL: test_correctly_analyzes_file (tools.fileinfo.features.max-memory.TestAnalysisWhenSufficient)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/tcagent/work/c1867d72f67fc407/retdec-regression-tests/tools/fileinfo/features/max-memory/test.py", line 43, in test_correctly_analyzes_file
    assert self.fileinfo.succeeded
AssertionError

----------------------------------------------------------------------
Ran 1 test in 0.001s

FAILED (failures=1)
```
Could you please take a look at it? Let me know if you are unable to reproduce the issue.

Reply:
For future reference: Resolved in https://github.com/avast/retdec-regression-tests/pull/78/commits/790897ce14ff68408db0068b0e0ce7a0c3ee8767 by increasing the memory limit.